tests/convergence_tests/poisson_mms_common.rs

```diff
@@ -80,26 +84,49 @@ where
     // for linear contractions
     let u = DVector::<f64>::zeros(mesh.vertices().len());
 
-    let vector_assembler = SerialVectorAssembler::<f64>::default();
+    // Node coloring for test of parallel assembler
+    let colors = color_nodes(mesh);
+
+    let vector_assembler = VectorAssembler::<f64>::default();
     let matrix_assembler = CsrAssembler::default();
 
-    let laplace_assembler = ElementEllipticAssemblerBuilder::new()
+    let source_assembler = ElementSourceAssemblerBuilder::new()
         .with_finite_element_space(mesh)
-        .with_operator(&LaplaceOperator)
+        // TODO: Use better quadrature
         .with_quadrature_table(&quadrature)
-        .with_u(&u)
+        .with_source(poisson_source_function)
         .build();
 
-    let mut a_global = matrix_assembler.assemble(&laplace_assembler)?;
+    let mut b_global = vector_assembler.assemble_vector(&source_assembler)?;
 
-    let source_assembler = ElementSourceAssemblerBuilder::new()
+    // Compare with parallel assembler
+    {
+        let par_vector_assembler = VectorParAssembler::<f64>::default();
+        let mut par_b_global = DVector::zeros(b_global.len());
+        par_vector_assembler.assemble_vector_into(&mut par_b_global, &colors, &source_assembler)?;
```

Comment:
Same here as for matrices: would be great if `VectorParAssembler` had a convenience `assemble_vector` method which would also allocate the output storage